fuzzers/libfuzzer_libpng_cmin/src/lib.rs

```diff
@@ -111,7 +114,7 @@ fn fuzz(corpus_dirs: &[PathBuf], objective_dir: PathBuf, broker_port: u16) -> Re
             // RNG
             StdRand::with_seed(current_nanos()),
             // Corpus that will be evolved, we keep it in memory for performance
-            InMemoryCorpus::new(),
+            CachedOnDiskCorpus::new(testcases_dir, 64).unwrap(),
```

Comment:
I wouldn't kick out the InMemoryCorpus personally. It should be enough to check the objective corpus in the end, or maybe parse the logs, as you suggested